File: 프로그래머스/1/42748. K번째수/K번째수.swift
import Foundation

func solution(_ array:[Int], _ commands:[[Int]]) -> [Int] {
    
    var tempArr:[Int] = []
    var answer:[Int] = []
    
    for i in 0..<commands.count {        
        tempArr = Array(array[commands[i][0] - 1...commands[i][1] - 1]).sorted()
        answer.append(tempArr[commands[i][2] - 1])
    }
    
    return answer
}

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {    
    var answer = [];    
    for(element of commands) {
        const arr = array.slice(element[0] - 1, element[1])
        const sortedArr = arr.sort((a,b) => a - b)        
        answer.push(sortedArr[element[2] - 1])        
    }    
    return answer;
}